.gitignore

```diff
@@ -5,3 +5,4 @@ django_picklefield.egg-info/*
 build/
 .coverage
 .tox
+*.sw[pon]
```

Comment:
Configure your local `~/.gitignore` file to ignore these files they have nothing to do with this project and are related to your own development environement.